THIRDPARTYNOTICES.md

```diff
@@ -234,7 +234,7 @@ This file is based on or incorporates material from the projects listed below (T
   > 
   >    END OF TERMS AND CONDITIONS
 
-## Hl7.Fhir.STU3 1.5.0
+## Hl7.Fhir.STU3 1.9.0
 * Component Source:   https://github.com/FirelyTeam/fhir-net-api
 * Component Copyright and License:  
   > Copyright (c) 2013-2019, HL7, Firely (info@fire.ly), Microsoft Open Technologies
```

Comment:
Should the copyright year be updated?

Reply:
Hmm.. I'm not familiar.  I will find out.

Reply:
Usually we copy whatever is on the license file.